index.css

```diff
@@ -0,0 +1,161 @@
+html,
+body {
+    margin: 0;
+    font-family: 'Karla', sans-serif;
+    background-color: #219ebc;
+}
+
+/*typography*/
+
+h1,
+h2,
+h3 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #EED6D3;
+    margin: 0;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+h2 {
+    font-size: 16px;
+}
+
+h3 {
+    margin: 25px;
+}
+
+/* layout */
+
+header {
+    background-color: #67595E;
+}
+
+.header-inner {
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    width: 370px;
+    gap: 15px;
+    padding: 10px 0 0 0;
+}
+
+.pumpkin-img {
+    width: 80px;
+}
+
+.controls-container {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    margin: 0px auto;
+}
+
+/* controls */
+
+.emotion-radios {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #bcbcbc;
+    border-radius: 5px;
+}
+
+.radio {
+    display: flex;
+    align-items: center;
+    border-bottom: 1px solid #bcbcbc;
+    padding: 4px 8px;
+}
+
+label {
+    font-size: 14px;
+    padding: 3px;
+    width: 100%;
+    cursor: pointer;
+}
+
+.highlight {
+    background-color: #fff0ee;
+    color: #bd301d;
+    accent-color: #bd301d;
+    font-weight: bold;
+}
+
+.radio:last-child {
+    border-bottom: 0;
+}
+
+input[type="radio"] {
+    cursor: pointer;
+}
+
+.gifs-check {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 16px;
+}
+
+.gifs-check-label {
+    width: unset;
```

Comment:
```suggestion
    width: auto;
```

Reply:
you're unsetting it, and auto is the default. Best just to set it to auto.

Reply:
Unsetting resets the width to the inherited value from the parent. What does width: auto, do?